gcd/src/main/java/org/spine3/server/storage/datastore/Kind.java

```diff
@@ -66,6 +81,14 @@ public static Kind of(TypeName typeName) {
         return new Kind(typeName.value());
     }
 
+    /**
+     * Produces a {@code Kind} representing the datastore namespace kind.
+     */
+    @Internal
+    public static Kind ofNamespace() {
```

Comment:
It's weird that this is `public`. Not a big deal for now, as it's internal. However, it may be a sign that the package is chosen incorrectly.